test/hotspot/jtreg/compiler/loopopts/superword/TestDependencyOffsets.java

```diff
@@ -23,7 +23,7 @@
 
 /*
  * @test id=vanilla-A
- * @bug 8298935 8308606 8310308 8312570 8310190
+ * @bug 8298935 8308606 8310308 8312570 8310190 8342387
```

Comment:
As an engineer in the Hotspot Compiler Team (Java Platform Group), Theo Weidmann needs access to the JDK source code, computing infrastructure and bug tracking system.